### ISSUE TITLE
Keep a track of object creation/update times

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#1764]
### Feature description
Keep a timestamp for when the object got:
- Created
- Last modified
It is possible to get a history of certain things this way through the framework

**Comment by wimpers** [comment]
@JeffreyDevloo  which objects do you mean? ALBA objects? In that case why would the framework be needed to store this information. Would it make more sense to store it in alba directly together with the object?

**Comment by JeffreyDevloo** [comment]
All of them. Storagerouters, storagedrivers, vdisks, osds, backends ...
It's some minor metadata that almost all databases keep track off, so why wouldn't we.

**Comment by wimpers** [comment]
Makes sense  to add this for what the Framework stores. Maybe pick it up during https://github.com/openvstorage/framework/issues/1179 ?

For ALBA objects I believe ALBA would be best suited to store that info. https://github.com/openvstorage/alba/issues/814
